app/static/tests/unit/components/wodinPanels.test.ts

```diff
@@ -0,0 +1,102 @@
+import { shallowMount, VueWrapper } from "@vue/test-utils";
+import { nextTick } from "vue";
+import WodinPanels from "../../../src/app/components/WodinPanels.vue";
+
+describe("WodinPaneks", () => {
+    const getWrapper = () => {
+        return shallowMount(WodinPanels, {
+            slots: {
+                left: "<div id=\"l-slot-content\">LEFT</div>",
+                right: "<div id=\"r-slot-content\">RIGHT</div>"
+            }
+        });
+    };
+
+    const testBothMode = (wrapper: VueWrapper) => {
+        const containerDiv = wrapper.find("div.wodin-mode-both");
+        expect(containerDiv.exists()).toBe(true);
+
+        expect(containerDiv.find("#collapse-left").exists()).toBe(true);
+        expect(containerDiv.find("#collapse-right").exists()).toBe(true);
+    };
+
+    const testRightMode = (wrapper: VueWrapper) => {
+        const containerDiv = wrapper.find("div.wodin-mode-right");
+        expect(containerDiv.exists()).toBe(true);
+
+        expect(containerDiv.find("#collapse-left").exists()).toBe(false);
+        expect(containerDiv.find("#collapse-right").exists()).toBe(true);
+    };
+
+    const testLeftMode = (wrapper: VueWrapper) => {
+        const containerDiv = wrapper.find("div.wodin-mode-left");
+        expect(containerDiv.exists()).toBe(true);
+
+        expect(containerDiv.find("#collapse-left").exists()).toBe(true);
+        expect(containerDiv.find("#collapse-right").exists()).toBe(false);
+    };
```

Comment:
These can be refactored to a method by passing `div.wodin-mode-whatever` as parameter.